Xamarin.Forms.Platform.Android/Renderers/AndroidGIFImageParser.cs

```diff
@@ -264,6 +257,50 @@ public static async Task<IFormsAnimationDrawable> LoadImageAnimationAsync(UriIma
    
 			return animation;
 		}
+
+		internal static async Task<FormsAnimationDrawable> GetFormsAnimationDrawableFromResource(int resourceId, Context context, BitmapFactory.Options options)
+		{
+			FormsAnimationDrawable animation = null;
+
+			using (var stream = context.Resources.OpenRawResource(resourceId))
+				animation = await GetFormsAnimationDrawableFromStream(stream, context, options);
+
+			return animation;
+		}
+
+		internal static async Task<FormsAnimationDrawable> GetFormsAnimationDrawableFromFile(string file, Context context, BitmapFactory.Options options)
```

Comment:
- Does this work on UWP?
- Does this work with a PNG? If I download a PNG/JPG to the file system to display it on Android does that currently work?

Reply:
I have applied changes and the sample works on Android, iOS and UWP.
![issue8821-uwp](https://user-images.githubusercontent.com/6755973/70799641-7bfc4e80-1daa-11ea-83f4-c12f3cc88d99.gif)